### PR TITLE
Update documentation for R

### PR DESCRIPTION
**Pull request by benkay86**
Now that we have R installed on the cluster, update the documentation to indicate we can simply module load R instead of having to learn conda. Also includes directions on setting up an R Jupyter kernel.

### CODE REVIEW

File: docs/software/r.md [hunk 1]
```diff
@@ -9,7 +9,87 @@ exclude: true
...
+
+Next, edit `~/.Rprofile` and add the following. Run `install.packages("Cairo")` if you have not already done so above. See [IRkernel #388](https://github.com/IRkernel/IRkernel/issues/388) for an explanation of why this step is needed.
+
+```
+## Set default 'type' for png() calls - useful when X11 device is not available!
+## NOTE: Needs 'cairo' capability
+options(bitmapType='cairo')
+```
+
+Then follow the [directions to start and connect to your Jupyter server](jupyter-nodebook.md). Before starting the server, make sure you `module load R` in your shell. You should see the following in your Jupyter launcher:
```
Comment (rcifadmin) [inline docs/software/r.md:86]: This should be s/nodebook/notebook/

**Review by rcifadmin — CHANGES_REQUESTED** [review]
Broken link - noted in comments (simple s/d/t/ req'd). Otherwise, LGTM!

File: docs/software/r.md (hunk 1 above)
Comment (benkay86) [reply docs/software/r.md]: Fixed, should be ready to merge.

**Review by rcifadmin — APPROVED** [review]
LGTM

**Pull request merged**